### ISSUE TITLE
Link to interface list when updating displaysets

### ISSUE THREAD

**Issue by chrisvanrun** [comic/grand-challenge.org#3555]
see title.

**Comment by amickan** [comment]
Not completely clear to me from the title. Do you mean include a link to the interface list on the DS create and update pages?

**Comment by chrisvanrun** [comment]
Yes, when creating a display set (not sure if batch or single) there is a help text with a link. It's missing in the hx rendered forms, had Loc confused when I was looking over his shoulder, creating a reader study.
